samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -121,7 +121,7 @@ The behavior of the commands vary depending on the hardware configuration and Kc
 
 * DTM with front-end module support:
 
-   * The official ``0x09`` DTM command when the :kconfig:option:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled, which is a default state.
+   * The official ``0x09`` DTM command when the :ref:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC <CONFIG_DTM_POWER_CONTROL_AUTOMATIC>` Kconfig option is enabled, which is a default state.
```

Comment:
I would expand it to something like:
```suggestion
   * When the :ref:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC <CONFIG_DTM_POWER_CONTROL_AUTOMATIC>` Kconfig option is enabled, which is a default state, the official ``0x09`` DTM command sets the final output power to the desired, or the closest available value by configuring the SoC output power and the front-end module gain.
```
But let's wait for Kamil so he can confirm.

Reply:
Looks good to me